client/jetbrains/webview/src/search/index.tsx

```diff
@@ -72,11 +73,9 @@ export function renderReactApp(): void {
                 <App
                     // Make sure we recreate the React app when the instance URL or access token changes to
                     // avoid showing stale data.
-                    key={`${instanceURL}-${accessToken}-${errorRetryIndex}`}
+                    key={`${instanceURL}-${getAuthenticatedUser()?.id}-${errorRetryIndex}`}
```

Comment:
looking at the code here I see that `getAuthenticatedUser()` can return `null` so eventually there is no `id` - am I correct? should we handle it?

Reply:
It doesn't really matter but I added fallback string to make it clear.